.github/contracts_tests.yaml

```
@@ -42,7 +42,7 @@ jobs:
       - uses: pnpm/action-setup@v4
         name: Install pnpm
         with:
-          version: 8
+          version: 9.8
```

```suggestion
          version: 9
```
Perhaps just use v9 and it will use latest of pnpm version 9

---

When using only major like this 9 it will always use the latest version.

The nice thing when running in CI like a workflow it will not change the package lock files.